src/components/Recipe.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { ContainerRecipe, Field } from '../styles/Recipe';
+import { IIngredient, IRecipe } from './../types/types'
+
+
+
+type RecipeProps = {
+    recipe: IRecipe,
+    size: string
+}
+
+function Recipe({ recipe, size }: RecipeProps) {
+  return (
+   <ContainerRecipe size={size}>
+             <h2>{recipe.name}</h2>
+      
+       <Field>
+          <h4>Description: </h4>
+          <div><p>{recipe.description}</p>  </div> 
+        </Field>
+        <Field>
+          <h4>Ingredients</h4>
+          <div>
+          {recipe.ingredients && recipe.ingredients.map((ingredient: IIngredient, index:number)=>{
```

Comment:
Little note on the formatting, I see it a bit off on Github, do you use a formatter in local?

Reply:
No I did not use a local formatter. I will use you now to clean up the formatting